cfme/tests/automate/test_domain.py

```diff
@@ -276,3 +277,40 @@ def test_domain_import_git(request, appliance, url, param_type, param_value, ver
     request.addfinalizer(domain.delete_if_exists)
     assert domain.exists
     domain.delete()
+
+
+@pytest.mark.tier(1)
+def test_object_attribute_type_in_automate_schedule(appliance):
+    """
+    Polarion:
+        assignee: ghubale
+        casecomponent: Automate
+        initialEstimate: 1/15h
+        startsin: 5.8
```

Comment:
Please change it to 5.9 or 5.10

Reply:
As per my understanding, ```startsin:``` is the version of cfme in which this feature introduced.

Reply:
Yes